TestsFoundation/TestsFoundation/DataSeeding/Conversation/DataSeeder+Conversation.swift

```diff
@@ -0,0 +1,43 @@
+//
+// This file is part of Canvas.
+// Copyright (C) 2023-present  Instructure, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+extension DataSeeder {
+    @discardableResult
+    public func createConversation(requestBody: CreateDSConversationRequest.RequestedDSConversation) -> DSConversation {
+        let request = CreateDSConversationRequest(body: requestBody)
+        return makeRequest(request)[0]
+    }
+
+    @discardableResult
+    public func updateConversation(conversationId: String, event: DSEvent) -> DSProgress {
+        let requestedBody = UpdateDSConversationRequest.Body(conversation_ids: [conversationId], event: event)
+        let request = UpdateDSConversationRequest(body: requestedBody)
+        return makeRequest(request)
+    }
```

Comment:
This is actually not in use right now in any of the test cases. I implemented this because of the archive stuff we talked about, but in the end it doesn't even matter... :)